flow/size.yaml

```diff
@@ -62,3 +83,107 @@ components:
+          default: imix
+          description: |-
+            Option allow to specify a predefined frame size distribution <size, weight> pairs (including IMIX distribution).  
+            The available predefined distribution pairs are:
+            - IMIX (64:7, 570:4, and 1518:1)  
+            - IPSec IMIX (90:58.67, 92:2, 594:23.66 and 1418:15.67)  
+            - IPv6 IMIX (60:58.67, 496:2, 594:23.66 and 1518:15.67)  
+            - Standard IMIX (58:58.67, 62:2, 594:23.66 and 1518:15.67)  
+            - TCP IMIX (90:58.67, 92:2, 594:23.66 and 1518:15.67)  
+            - RPR Trimodal (64:60, 512:20 and 1518:20)  
+            - RPR Quadmodal (64:50, 512:15, 1518:15 and 9216:20)  
+            - Cisco (64:7, 594:4 and 1518:1)  
+            - Tolly (64:55, 78:5, 576:17 and 1518:23)  
+          x-field-uid: 2
+          x-enum:
+            imix:
+              x-field-uid: 1
+            ipsec_imix:
+              x-field-uid: 2
+            ipv6_imix:
+              x-field-uid: 3
+            standard_imix:
+              x-field-uid: 4
+            tcp_imix:
+              x-field-uid: 5
+            rpr_trimodal:
+              x-field-uid: 6
+            rpr_quadmodal:
+              x-field-uid: 7
+            cisco:
+              x-field-uid: 8
+            tolly:
+              x-field-uid: 9
+        custom:
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.SizeWeightPairs.Custom'
+          x-field-uid: 3
+    Flow.SizeWeightPairs.Custom:
+      type: object
+      description: |-
+        Option allow to set a custom frame size distribution <size, weight> pairs.
+      properties:
+        size:
+          description: The size of the frame (in bytes) for this weight pair.
+          type: integer
+          minimum: 12
+          maximum: 65535
+          default: 64
+          x-field-uid: 1
+        weight:
+          description: A user-defined, 32-bit integer (weight) assigned to the corresponding frame size in this weight pair. 
+          type: integer
+          format: uint32
+          minimum: 1
+          default: 1
+          x-field-uid: 2
+    Flow.Size.Gaussian:
+      type: object
+      description: |-
+        Gaussian distribution.
+      properties:
+        center:
+          description: |-
+            The frame size for the center of the distribution curve.
+          type: number
+          format: double
+          minimum: 10
+          maximum: 13312
+          default: 200.00
+          x-field-uid: 1
+        width_at_half:
+          description: |-
+            The width of the distribution curve at half of the curve weight, expressed as 
+            the difference between the minimum and maximum frame size in bytes.
+          type: number
+          format: double
+          minimum: 0.01
+          default: 100.00
+          x-field-uid: 2
+        weight:
+          description: |-
+            The priority to give the distribution curve.
```

Comment:
Again, good to specify what it would mean for priority to be higher or lower? Same priority would mean all 4 flows would have same representation and high priority would mean that more of the packets would conform to the distribution with higher priority?

Reply:
Explained with image reference.